contracts/pcv/morpho/MorphoCompoundPCVDeposit.sol

```diff
@@ -0,0 +1,119 @@
+/// surfaced in Morpho and liquidity is locked, Volt will need to rely on social
+/// coordination with the Morpho team to recover funds.
+contract MorphoCompoundPCVDeposit is PCVDeposit {
+    using SafeERC20 for IERC20;
+
+    /// @notice reference to the lens contract for morpho-compound v2
+    address public constant LENS = 0x930f1b46e1D081Ec1524efD95752bE3eCe51EF67;
+
+    /// @notice reference to the morpho-compound v2 market
+    IMorpho public constant MORPHO =
+        IMorpho(0x8888882f8f843896699869179fB6E4f7e3B58888);
+
+    /// @notice reference to underlying token
+    address public immutable token;
+
+    /// @notice cToken in compound this deposit tracks
+    /// used to inform morpho about the desired market to supply liquidity
+    address public immutable cToken;
+
+    constructor(
+        address _core,
+        address _cToken,
+        address _underlying
+    ) CoreRef(_core) {
+        cToken = _cToken;
+        token = _underlying;
+    }
+
+    /// @notice Returns the distribution of assets supplied by this contract through Morpho-Compound.
+    /// @return sum of suppliedP2P and suppliedOnPool for the given CToken
+    function balance() public view override returns (uint256) {
+        (, , uint256 totalSupplied) = ILens(LENS).getCurrentSupplyBalanceInOf(
+            cToken,
+            address(this)
+        );
+
+        return totalSupplied;
+    }
+
+    /// @notice returns the underlying token of this deposit
+    function balanceReportedIn() external view returns (address) {
+        return token;
+    }
+
+    /// @notice deposit ERC-20 tokens to Morpho-Compound
+    function deposit() public whenNotPaused {
+        uint256 amount = IERC20(token).balanceOf(address(this));
+        if (amount == 0) {
+            /// no op to prevent revert on empty deposit
+            return;
+        }
+
+        IERC20(token).approve(address(MORPHO), amount);
+        MORPHO.supply(
+            cToken, /// cToken to supply liquidity to
+            address(this), /// the address of the user you want to supply on behalf of
+            amount
+        );
+
+        emit Deposit(msg.sender, amount);
+    }
+
+    /// @notice withdraw tokens from the PCV allocation
+    /// @param to the address PCV will be sent to
+    /// @param amount of tokens withdrawn
+    function withdraw(address to, uint256 amount) external onlyPCVController {
+        _withdraw(to, amount);
+    }
+
+    /// @notice withdraw all tokens from Morpho
+    /// @param to the address PCV will be sent to
+    function withdrawAll(address to) external onlyPCVController {
+        uint256 amount = balance();
+        _withdraw(to, amount);
+    }
+
+    /// @notice helper function to avoid repeated code in withdraw and withdrawAll
+    function _withdraw(address to, uint256 amount) private {
+        MORPHO.withdraw(cToken, amount);
+        IERC20(token).safeTransfer(to, amount);
+
+        emit Withdrawal(msg.sender, to, amount);
+    }
```

Comment:
This is what `withdraw` with proper profit accounting could look like: 

```suggestion
    function _withdraw(address to, uint256 amount) private {
        // compute profit from interests and emit an event
        uint256 _depositedAmount = depositedAmount; // SLOAD
        uint256 _balance = balance();
        uint256 profit = _balance - _depositedAmount;
        emit Harvest(address(token), int256(profit), block.timestamp);
    
        MORPHO.withdraw(cToken, amount);
        IERC20(token).safeTransfer(to, amount);
        
        // update tracked deposit amount
        depositedAmount = _balance - amount;

        emit Withdrawal(msg.sender, to, amount);
    }
```

It adds a few lines, so maybe add a "simpleWithdraw()" method that just exists Morpho